### DISCUSSION TITLE
overly aggresive typescript promotion

### DISCUSSION THREAD

**Discussion by mark-hahn** [microsoft/vscode-discussions#2423] [Extension Show and Tell]
I'm getting this ...

For <template lang="pug">, the @vue/language-plugin-pug plugin is required. Install it using $ npm install -D @vue/language-plugin-pug and add it to vueCompilerOptions.plugins to support TypeScript intellisense in Pug templates.

It says `plugin is required` but then says `to support TypeScript intellisense`.  I have no desire for anything typescript related.  How do I get rid of this "error"?

**Comment by isidorn** [comment]
This forum is for VS Code extension authors. For general user questions please go to stack overflow.
